CHANGES.md

```diff
@@ -38,6 +40,9 @@ To be released.
 ### Added APIs
 
  -  Added `DefaultStore` class to replace `LiteDBStore`.  [[#662]]
+ -  Added `BlockChain<T>(IBlockPolicy<T>, IStore, Block<T>)`.
+    [[#688]]
```

Comment:
It seems redundant as you already added an item to the *Backward-incompatible API changes* section.